packages/client/tests/functional/issues/13766/prisma/_schema.ts

```diff
@@ -11,25 +11,21 @@ export default testMatrix.setupSchema(({ provider }) => {
     datasource db {
       provider = "${provider}"
       url      = env("DATABASE_URI_${provider}")
-    }
-
-    enum OrderStatus {
-      NEW
-      PICKED
-      STORED
-      DELIVERED
```

Comment:
Just cleanup because irrelevant or did this cause trouble @Jolg42?

Reply:
To make it possible to run SQLite and SQL Server
https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#enum
>Not supported by SQLite and Microsoft SQL Server
The [SQLite connector](https://www.prisma.io/docs/concepts/database-connectors/sqlite) and the The [Microsoft SQL Server connector](https://www.prisma.io/docs/concepts/database-connectors/sql-server) do not support the enum type.